meerk40t/gui/navigationpanels.py

```diff
@@ -487,6 +527,59 @@ def on_button_align_trace_quick(
         self.context("trace_quick\n")
         self.drag_ready(True)
 
+    def pane_show(self, *args):
+        self.context.listen("driver;position", self.on_update)
+        self.context.listen("emulator;position", self.on_update)
+
+    # Not sure whether this is the right thing to do, if it's still locked and then
+    # the pane gets hidden?! Let's call it a feature for now...
```

Comment:
Pane hide is also called at the close of the program to unhook those things. Mostly as a sanity check.